schemas/input/csl-data.json

```
@@ -308,6 +309,10 @@
         "type": ["string", "number"]
       },
       "event": {
+        "description": "Alias for 'event-title'. Deprecated. Will be removed in CSL 1.1.",
```

```suggestion
        "description": "[Deprecated - use 'event-title' instead. Will be removed in 1.1]",
```

The above is a convention used in the documentation tool I was planing to use.